src/codegen.cpp

```diff
@@ -72,6 +72,7 @@ CodeGen *codegen_create(Buf *root_source_dir, const ZigTarget *target) {
     g->root_package = new_package(buf_ptr(root_source_dir), "");
     g->std_package = new_package(ZIG_STD_DIR, "index.zig");
```

Comment:
hmm I just realized this PR is not quite this simple because `g->std_package` depends on `ZIG_STD_DIR` as well. I'll need to move some code around to allow overwriting the standard dir at runtime. It can be done, I'll take care of it from here.